script.js

```diff
@@ -1,4 +1,124 @@
+// store game modes
+var gameMode = "start";
+var DICEROLLMODE = "dice roll mode";
+var ORDERMODE = "order mode";
+var ENTERNUMOFUSERS = "enter number of users";
+
+// arrays
+
+var allDiceRollsArray = [];
+var orderedNumberArray = [];
+var orderedNumberArrayforSorting = [];
+
+//counts
+var numOfPlayers = 0;
+var playerNumber = 1;
+var numDiceRollsOverall = 0;
+var diceRollsPerUser = 0;
+var numOfUsersWhoHavePlayed = 0;
```

Comment:
How does this variable relate to `numOfPlayers`? Could we use the same variable instead of having to track 2 variables?